### PR TITLE
Update how-to-install-macos-client.md

### PR DESCRIPTION
**Pull request by Appelcloud**
Added M4 processor in the prerequisites part of Documentation

### CODE REVIEW

**Comment by ttorble** [comment]
@HULKsmashGithub 

Can you review the proposed changes?  

IMPORTANT: When the changes are ready for publication, adding a `#sign-off` comment is the best way to signal that the PR is ready for the review team to merge.  

#label:"aq-pr-triaged" 
@MicrosoftDocs/public-repo-pr-review-team

File: docs/global-secure-access/how-to-install-macos-client.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ This article describes how to download and install the Global Secure Access clie
 
 ## Prerequisites
 
-- A Mac device with an Intel, M1, M2, or M3 processor, running macOS version 13 or newer.
+- A Mac device with an Intel, M1, M2, M3 or M4 processor, running macOS version 13 or newer.
```
Comment (HULKsmashGithub) [inline docs/global-secure-access/how-to-install-macos-client.md:22]:
```suggestion
- A Mac device with an Intel, M1, M2, M3, or M4 processor, running macOS version 13 or newer.
```

**Comment by HULKsmashGithub** [comment]
#sign-off

**Comment by HULKsmashGithub** [comment]
#sign-off

**Pull request merged**